EIPS/eip-1001.md

```diff
@@ -40,6 +40,10 @@ Where all of the key + value pairs are optional, allowing for maximum flexibilit
 `name` (optional) is a name ofthe private key - e.g. "paper wallet"
 `type` (optional) is the type of key (STRING) - Defaults to ECDSA
 
+### Security Warning
+
+Since private keys are highly sensitive information, it is considerably safer if input (via QR code, keyboard etc.) is handled directly by the target application, rather than going through some IPC mechanism (e.g. the Intent mechanism in Android OS), trusting third-party applications (such as a QR code reader) with the private key. Thus, it is **recommended** to display a security warning, whenever the application receives a private key through IPC messaging, warning the user about the risks associated with using a third-party application to input private keys.
```

Comment:
Thanks! Just not sure about the mention of keyboard here. Keyboard opens a big can of worms on android and I also do not think humans should enter their private keys via keyboard. Let's discuss this in person later.